extensions/wikia/CreateNewWiki/classes/Starters.class.php

```diff
@@ -12,6 +12,8 @@
 class Starters {
 
 	const STARTER_DUMPS_BUCKET = 'starter';
+	// TODO Clean up in XW-2393
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)